privacyidea/lib/utils/__init__.py

```diff
@@ -1480,3 +1481,41 @@ def parse_string_to_dict(s, split_char=":"):
     values = [[x for x in y.split()] for y in packed_list[1::2]]
     d = {a: b for a, b in zip(keys, values)}
     return d
+
+
+def replace_function_event_handler(text, token_serial=None, tokenowner=None, logged_in_user=None):
+    if logged_in_user is not None:
+        login = logged_in_user.login
+        realm = logged_in_user.realm
+    else:
+        login = ""
+        realm = ""
+
+    if tokenowner is not None:
+        surname = tokenowner.info.get("surname")
+        givenname = tokenowner.info.get("givenname")
+        userrealm = tokenowner.realm
+    else:
+        surname = ""
+        givenname = ""
+        userrealm = ""
+
+    if token_serial is not None:
```

Comment:
You can shorten this to 
`token_serial = token_serial if token_serial else ""`

Reply:
604e4e6